README.md

```diff
@@ -83,21 +98,18 @@ iex> {:ok, channel} = GRPC.Stub.connect("localhost:50051", interceptors: [GRPC.L
 
 Check [examples](examples) and [interop](interop)(Interoperability Test) for some examples.
 
-## TODO
+## Features
```

Comment:
Given that all the items on the list are done, I think it makes more sense to turn this section into a list of implemented features.

Reply:
I would remove this section altogether instead.
Or add a less itemized version in its place

Reply:
> add a less itemized version in its place
I agree with you. What kind of items can we keep or what features are important to mention in the documentation? @polvalente

Reply:
> > add a less itemized version in its place
> 
> I agree with you. What kind of items can we keep or what features are important to mention in the documentation? @polvalente

Supported request types (unary, stream in, stream out, bid stream), encoding types (grpc+web, grpc. etc), transport types (uds, tcp, tls), grpc reflection (in progress) , http transcoding I think are important things

Reply:
> I wouldn't add anything that's still in progress. Otherwise, I agree with @sleipnir

It was not my intention to say to put whatever is in progress. Just present an overview of what are the interesting features of any grpc library.